server/sampleData/utils/helpers.ts

```diff
@@ -104,10 +106,12 @@ export const loadSampleData = (
           body: bulkBody,
         });
         if (resp.errors) {
-          console.log('Error while bulk inserting. ', resp.errors);
-          return Promise.reject(
-            new Error('Error while bulk inserting. Please try again.')
-          );
+          const errorItems = resp.items;
+          const firstErrorReason = isEmpty(errorItems)
+            ? 'Error while bulk inserting. Please try again.'
+            : prettifyErrorMessage(errorItems[0].index.error.reason);
+          console.log(firstErrorReason);
```

Comment:
minor: change to `console.error` or remove

Reply:
sure. good catch. will remove it.